cmd/gaia/init/utils.go

```diff
@@ -22,16 +20,13 @@ import (
 
 // ExportGenesisFile creates and writes the genesis configuration to disk. An
 // error is returned if building or writing the configuration to file fails.
-func ExportGenesisFile(
+func ExportGenesisFile(genDoc *types.GenesisDoc,
```

Comment:
this function signature is strange... if we're passing in genDoc, there's no need to pass in chainID, validators, or appState.

Reply:
addressed